src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java

```diff
@@ -25,6 +28,16 @@ public void triggerOnScrollSlots() {
     jsActions.scrollToBottom();
   }
 
+  public void fixScrollPositionByNavbarOnF2(boolean isMobile) {
+    if(isMobile){
```

Comment:
Both cases behind `if` have common logic, you are using `if` statement only to determine which element use, so let's refactor it

Reply:
Done fixed